.github/workflows/build.yml

```diff
@@ -43,6 +43,20 @@ jobs:
             cxx-compiler: clang++
             cc-compiler: clang
             compiler-desc: Clang
+          - name_prefix: Linux CentOS 7 VFX CY2023
+            release_prefix: linux-vfx2023
+            ostype: linux
+            distro: centos7
+            os: ubuntu-latest
```

Comment:
This is also present for the 2021 and 2022 aswf-docker containers, and seems to get written to a `OSNAME` env var.

Reply:
Not sure what fix this comment is suggesting. I'm not sure OSNAME is used, but not worth removing at the moment. The `os` itself is used to select the OS to run the CI build on.

Reply:
I think this comment is obsolete, your second commit got rid of the confusing `distro: centos7` label which was confusing me with `os: ubuntu-latest` but of course the `os:` label is for the runner itself, and the `distro:` was meant for what's inside the container running the build.